src/studio-home/tabs-section/TabsSection.test.tsx

```diff
@@ -308,24 +311,37 @@ describe('<TabsSection />', () => {
       )).toBeVisible();
     });
 
-    it('should switch to Libraries tab and render specific v1 library details ("v1 only" mode)', async () => {
-      render({ librariesV2Enabled: false });
+    it('should switch to Libraries tab and render specific v1 library details - v1 only mode', async () => {
       await axiosMock.onGet(getStudioHomeApiUrl()).reply(200, generateGetStudioHomeDataApiResponse());
       await axiosMock.onGet(libraryApiLink).reply(200, generateGetStudioHomeLibrariesApiResponse());
-      await executeThunk(fetchStudioHomeData(), store.dispatch);
-      await executeThunk(fetchLibraryData(), store.dispatch);
+      render({ librariesV2Enabled: false });
+      const user = userEvent.setup();
+      await act(async () => executeThunk(fetchStudioHomeData(), store.dispatch));
 
       // Libraries v2 tab should not be shown
       expect(screen.queryByRole('tab', { name: librariesBetaTabTitle })).toBeNull();
 
       const librariesTab = await screen.findByRole('tab', { name: tabMessages.librariesTabTitle.defaultMessage });
-      fireEvent.click(librariesTab);
+      await user.click(librariesTab);
```

Comment:
Out of curiosity, did the `userEvent` work here better than the `fireEvent`? Would you happen to know why?

Reply:
I didn't know that it was the recommended way! Thanks!